### DISCUSSION TITLE
扫码获取stoken返回3503

### DISCUSSION THREAD

**Discussion by firetail-zhy** [UIGF-org/mihoyo-api-collect#72] [Q&A]
https://github.com/UIGF-org/mihoyo-api-collect/blob/main/hoyolab/login/qrcode_hk4e.md#%E6%9F%A5%E8%AF%A2%E4%BA%8C%E7%BB%B4%E7%A0%81%E6%89%AB%E6%8F%8F%E7%8A%B6%E6%80%81

返回json
{
    "data": null,
    "message": "请求失败，当前设备或网络环境存在风险",
    "retcode": -3503
}

**Comment by SC0624a** [comment]
我也遇到了这个问题，想问问怎么解决

**Reply by MuxiaoWF** [reply]
似乎把app_id换成其他的游戏就好了（比如未定的2），可能是测试的时候刷太多次同一个游戏导致的？

**Comment by moyanj** [comment]
查询二维码扫描状态的device_id要和申请二维码的一样

**Comment by ShikiSuen** [comment]
这个问题已经无法用崩二学员与未定事件簿的 appID 绕过了，会出现这个故障：
https://github.com/pizza-studio/PizzaHelperUnited/issues/227
米游社在扫码之后点击确认按钮之后弹出 NSAlert `decode err response body error: unexpected end of JSON input`。
